packages/network-controller/src/NetworkController.ts

```diff
@@ -788,9 +786,9 @@ function validateNetworkControllerState(state: NetworkState) {
 
   if (!networkClientIds.includes(state.selectedNetworkClientId)) {
     throw new Error(
-      `NetworkController state is invalid: \`selectedNetworkClientId\` ${inspect(
-        state.selectedNetworkClientId,
-      )} does not refer to an RPC endpoint within a network configuration`,
+      // False negative - this is a string.
```

Comment:
Huh. I can reproduce this locally as well. Strange bug.

I don't see any reports of this on the `@typescript-eslint` repo. Maybe this is caused by us using an incompatible version of TypeScript though.

Reply:
Yeah, I'd be curious how the situation is once we upgrade.